pytest_httpx/_httpx_mock.py

```diff
@@ -169,13 +169,19 @@ def _handle_request(
         self._requests.append(request)
 
         response = self._get_response(request)
+        if not response:
+            callback = self._get_callback(request)
+            if callback:
+                response = callback(request)
```

Comment:
The only difference is that, now, we will not return anything returning `False` as truth value. But I guess that's ok because it should always be a response anyway, pretty sure httpx would raise an exception in this case otherwise :)